panoptes_client/caesar.py

```diff
@@ -0,0 +1,126 @@
+from panoptes_client.panoptes import Panoptes
+
+
+class Caesar(object):
+    EXTRACTOR_TYPES = ['blank', 'external', 'question', 'survey', 'who', 'pluck_field', 'shape']
+    REDUCER_TYPES = ['consensus', 'count', 'placeholder', 'external', 'first_extract', 'stats', 'unique_count', 'rectangle', 'sqs']
+    RULE_TO_ACTION_TYPES = {
+        'subject': ['retire_subject', 'add_subject_to_set', 'add_to_collection', 'external'],
+        'user': ['promote_user']
+    }
+
+    def __init__(self, endpoint='https://caesar-staging.zooniverse.org', redirect_url='https://caesar.zooniverse.org/auth/zooniverse/callback'):
+        self.endpoint = endpoint
+        self.headers = {
+            'Accept': 'application/json'
+        }
+
+    def http_get(self, *args, **kwargs):
+        kwargs['endpoint'] = self.endpoint
+        kwargs['headers'] = self.headers
```

Comment:
Not blocking - could these args be passed in via the `http_get` method as well to allow overriding at the call site?  e.g. allow the endpoint to be changed...
```
kwargs['endpoint'] ||= self.endpoint
```